kong/runloop/plugin_servers/process.lua

```diff
@@ -150,16 +151,19 @@ local function pluginserver_timer(premature, server_def)
     end
 
     kong.log.notice("[pluginserver] starting pluginserver process for ", server_def.name or "")
-    server_def.proc = assert(ngx_pipe.spawn("exec " .. server_def.start_command, {
```

Comment:
the `server_ref` passed in here is the reference of `kong.configuration.pluginservers`, so the `proc`(which is just a runtime sub process handle) here will be included in configuration, which is not reasonable, and also causes  encoding of configuration failed on admin API GET method.